### ISSUE TITLE
Trained Mask RCNN without ground truth bounding boxes 

### ISSUE THREAD

**Issue by JasperShian** [pytorch/vision#1553]
Hi all,

Is acceptable to train mask rcnn without bounding boxes? I want to generate only negative samples after RPN model in order to lower false positive cases.

**Comment by fmassa** [comment]
The current implementation does not support training without positive boxes.
But you'll need at least some positive boxes for training to make sense.
